types/puppeteer/index.d.ts

```diff
@@ -38,29 +38,29 @@ export interface Dialog {
   defaultValue(): string;
   dismiss(): Promise<void>;
   message(): string;
-  type: "alert" | "beforeunload" | "confirm" | "prompt";
+  type: 'alert' | 'beforeunload' | 'confirm' | 'prompt';
 }
 
 export type PageEvents =
-  | "console"
-  | "dialog"
-  | "error"
-  | "frameattached"
-  | "framedetached"
-  | "framenavigated"
-  | "load"
-  | "pageerror"
-  | "request"
-  | "requestfailed"
-  | "requestfinished"
-  | "response";
+  | 'console'
```

Comment:
That's quite a lot of unnecessary diff noise here. Perhaps your editor automatically changed these lines?

Reply:
It did, I'll convert these back.